modules/backend/formwidgets/mediafinder/partials/_mediafinder.php

```diff
@@ -7,17 +7,15 @@
     <?php
     switch ($mode) {
         case 'image':
-            echo $this->makePartial('image_single');
-            break;
         case 'video':
         case 'audio':
```

Comment:
What will the preview be for an audio file? Was this missed here?

Reply:
Ah, I do intend to add an audio player to the backend field preview if an audio file is selected.